### DISCUSSION TITLE
Actividad 0 - Catelli, Renata Victoria

### DISCUSSION THREAD

**Discussion by renatacatelli** [frlp-utn-ingsoft/catedra#20] [Show and tell]
https://github.com/renatacatelli/ING-SOFTWARE

**Comment by RodrigoJacznik** [comment]
Hola.
Buenos mensajes de commits.
Lo quise correr pero creo que te falto pushear la carpeta templates y no lo pude usar. Por favor arreglalo para aprobar la actividad.

Otro detalle, en el settings.py, en el nombre de la base de datos (linea 80 de https://github.com/renatacatelli/ING-SOFTWARE/blob/master/mysite/mysite/settings.py) podes usar `BASE_DIR / "db.sqlite3"` para que funcione en cualquier lado, no solo en tu compu.

Saludos.

**Reply by renatacatelli** [reply]
Hola! La carpeta templates que tenía el index y el detail había un paso que decía de borrarlos y tener las vistas en mysite/polls/views. ¿Puede ser? ¿O hay que rehacer los archivos de templates? Y perfecto, ahora corrijo lo de la base de datos. Gracias!

**Reply by RodrigoJacznik** [reply]
En que parte dice que hay que borrar la carpeta templates? Puede ser que sea un error de traduccion.

**Reply by renatacatelli** [reply]
Lo habia visto acá pero lo entendí mal yo:

![image](https://github.com/frlp-utn-ingsoft/catedra/assets/103262589/0fe40982-7559-4957-a261-7aa5fcaa5454)

Recien los volví a agregar los templates pero no me deja correr el servidor para probarlo porque me sale un error en la bd pero me resulta raro porque no modifiqué nada de ahí salvo la corrección que me hiciste:

![image](https://github.com/frlp-utn-ingsoft/catedra/assets/103262589/e27225a5-3c26-4eaa-a94e-650ae36da50e)

**Reply by RodrigoJacznik** [reply]
Las comillas simples en la línea 80 del archivo settings.py no van.
Cambia `'NAME': 'BASE_DIR / "db.sqlite3"'` por         `NAME': 'BASE_DIR / "db.sqlite3"`,

**Reply by renatacatelli** [reply]
Hola, gracias! Ya lo corregi y lo subí. También pude probar la app.

**Reply by RodrigoJacznik** [reply]
Bien, ahora lo puede correr. Lo único que las `polls` que están en la db que pusheaste no tienen `choice` te olvidaste de pushearla?
Una `poll` se tendría que ver así:
![image](https://github.com/frlp-utn-ingsoft/catedra/assets/2715743/6198e118-075c-4cb2-a737-ad2278f453ee)
Creo que te falto la ultima parte de esta sección https://docs.djangoproject.com/en/5.0/intro/tutorial02/#playing-with-the-api, la que esta justo antes de "Introducing the Django Admin".

En la parte 7 (https://docs.djangoproject.com/en/5.0/intro/tutorial07/) se muestra como modificar el admin para agregar `choice` a una `poll`. Esta parte no es obligatorio que lo hagas pero quizás es mas fácil que agregar las `choice` mediante la shell de django.

**Reply by renatacatelli** [reply]
Gracias! Lo había hecho el jueves esa parte pero creo que me pasé algunas lineas. Recién volví a probarlo y pushearlo.

**Comment by RodrigoJacznik** [comment]
Genial.
La actividad esta aprobada.
